src/oauth2/client.cr

```diff
@@ -165,15 +182,12 @@ class OAuth2::Client
     end
   end
 
-  private def token_uri
-    uri = URI.parse(@token_uri)
-
+  private def get_token_uri : URI
```

Comment:
What's the point of this rename?

Reply:
Good question ;-)

Will switch it back.